Makefile

```diff
@@ -76,6 +76,26 @@ package-dashboard-rpm:
 
 	rm -rf ${PWD}/build
 
+### build rpm for apisix-openresty:
+.PHONY: package-apisix-openresty-centos7-rpm
+package-apisix-openresty-centos7-rpm:
+	./build-apisix-openresty-centos7.sh
+	fpm -f -s dir -t rpm \
+		-n apisix-openresty \
+		-a `uname -i` \
+		-v $(version) \
+		--iteration $(iteration) \
+		-d 'openresty-openssl111' \
```

Comment:
I've already installed `openresty-openssl111-devel` ,`openresty-pcre-devel`, `openresty-zlib-devel` since I built the rpm, do I still need to specify them here?